sklearn/utils/_fake_openmp.h

```diff
@@ -0,0 +1,13 @@
+typedef int omp_lock_t;
```

Comment:
instead of `fake` maybe `mock` is more intuitive.

Reply:
I removed it because it actually made things more complex instead of more simple as I hoped